### DISCUSSION TITLE
Has anyone tried integrating rqlite?

### DISCUSSION THREAD

**Discussion by Zacama** [ory/hydra#3583] [Ideas]
Has anyone tried integrating [rqlite](https://github.com/rqlite/rqlite)? Use an in-memory database to improve performance when hardware resources are limited and set some persistence strategies to ensure that data will not be lost.

**Comment by vinckr** [comment]
Hello @Zacama 

AFAIK you are the first to consider this! 

The usecase would be in minimal deploys where resources are very limited?

**Comment by Zacama** [comment]
Hello @vinckr 
Deploying a large-scale mysql cluster to achieve high-performance hydra requires a lot of cost. In K8s, if multi-copy hydra can use a distributed memory database or connect to a software that can serve as a cache layer for an existing distributed database, then the performance of hydra can be significantly improved under limited hardware resources.
